safe/engine/test/test_engine.py

```diff
@@ -414,6 +418,45 @@ def test_polygon_hazard_with_holes_and_raster_exposure(self):
 
     test_polygon_hazard_with_holes_and_raster_exposure.slow = True
 
+    def test_user_directory_when_saving(self):
+        # These imports must be inside the test.
+        from PyQt4.QtCore import QCoreApplication, QSettings
+        from qgis.core import QgsApplication
+
+        # noinspection PyCallByClass,PyArgumentList
+        QCoreApplication.setOrganizationName('QGIS')
+        # noinspection PyCallByClass,PyArgumentList
+        QCoreApplication.setOrganizationDomain('qgis.org')
+        # noinspection PyCallByClass
+        QCoreApplication.setApplicationName('QGIS2InaSAFETesting')
```

Comment:
@Gustry YEs best practice would be for files / paths created by InaSAFE to not include accents (unless we are propagating a name defined by the user).